mem_storage/src/main/java/com/splicemachine/storage/MPartitionFactory.java

```diff
@@ -101,6 +101,11 @@ public PartitionCreator withSplitKeys(byte[][] splitKeys) {
             return this;
         }
 
+        @Override
+        public PartitionCreator withCatalogVersion(String version) {
+            //no-op
```

Comment:
I would throw, this could be misleading for users.

Reply:
+1